Doc/library/http.server.rst

```diff
@@ -419,6 +426,16 @@ such as using different index file names by overriding the class attribute
 :attr:`index_pages`.
 
 
+By default, the server uses the MIME type ``'application/octet-stream'`` for
+the ``Content-Type`` header when the content type cannot be guessed from the
+URL's extension (if any). The option ``--default-content-type`` overrides
+this default with one of the user's choosing::
+
+        python -m http.server --default-content-type text/html
+
+.. versionchanged:: next
+   Added the ``--default-content-type`` option.
```

Comment:
```suggestion
.. option:: -default-content-type <content_type>

   Specifies the Content-Type HTTP header. By default, the server uses the
   MIME type ``'application/octet-stream'`` for the ``Content-Type`` header
   when the content type cannot be guessed from the URL's extension (if any)::

        python -m http.server --default-content-type text/html

   .. versionadded:: next
```

Reply:
we added usage of `.. option::` directory for CLI flags, we also need to move the description of this flag to the right place, could you move it? (I also forgot the second `-` before `default-content-type`, please add it :)